src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/LayoutPath.java

```diff
@@ -25,8 +25,10 @@
  */
 package jdk.internal.foreign;
 
+import jdk.incubator.foreign.MemoryAddress;
 import jdk.incubator.foreign.MemoryHandles;
 import jdk.incubator.foreign.MemoryLayout;
+import jdk.incubator.foreign.MemoryLayouts;
```

Comment:
The new imports seem to be unused
```suggestion
import jdk.incubator.foreign.MemoryHandles;
import jdk.incubator.foreign.MemoryLayout;
```